inlong-sort/sort-common/src/test/java/org/apache/inlong/sort/protocol/node/load/ClickHouseLoadNodeTest.java

```diff
@@ -44,6 +44,11 @@ public Node getTestObject() {
                 "ck_demo",
                 "jdbc:clickhouse://localhost:8023/default",
                 "root",
-                "root");
+                "root",
+                "Log",

```

Comment:
The `Log` was one of the engines of ClickHouse?

Reply:
yes, [https://clickhouse.com/docs/en/engines/table-engines/log-family/log/](https://clickhouse.com/docs/en/engines/table-engines/log-family/log/)